README.md

```diff
@@ -1,3 +1,48 @@
 # MiniProxy [![Build Status](https://travis-ci.org/conversation/miniproxy.svg?branch=master)](https://travis-ci.org/conversation/miniproxy)
 
-It's a thing!
+A small stubbable proxy server for testing HTTP(S) interactions.
+
+
+## Getting Started
+
+In your Gemfile:
+
+    gem 'miniproxy'
+
+Configure Capybara (via Chrome and Selenium) to route all requests through the server (`spec/support/capybara_driver.rb`):
+
+    chrome_options = Selenium::WebDriver::Chrome::Options.new
+    chrome_options.add_argument("--proxy-server=#{MiniProxy::Server.host}:#{MiniProxy::Server.port}")
+
```

Comment:
It's probably also worth including the 
```
chrome_options.add_argument("--ignore-certificate-errors")
```
line here, otherwise https mocks won't work.

It is imperfect though, because it (silently) doesn't work in conjunction with `--headless`...